docs/generated/cli/workspace-lint.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: 'workspace-lint - CLI command'
-description: 'Lint nx specific workspace files (nx.json, workspace.json) and check existence of the configured packages and apps.  Note: To exclude files from this lint rule, you can add them to the `.nxignore` file'
+description: "Lint nx specific workspace files (nx.json, workspace.json), checks if your configuration and your source code are in sync. This will identify any projects with no files in the configured project root folder, as well as any file that's not part of any project configured in the workspace. Note: To exclude files from this lint rule, you can add them to the `.nxignore` file"
```

Comment:
```
This will identify any projects with no files in the configured project root folder, as well as any file that's not part of any project configured in the workspace.
```

I think the above has too many mentions of "any", and there is inconsistency between  `any projects` and `any file` where the first is plural but second is singular. 

Maybe something like this is better?


```
This command identifies projects with no files in the configured project root folder, as well as files that are not part of any projects in the workspace.
```

Reply:
Yup much better indeed! Thank you!